ONSdigital/eq-deploy
# Author Jenkins Jobs

**_What**_
The new Jenkins jobs for the Author application; i.e. Elastic beanstalk deploy and the two terraform jobs. I've also added the missing Destroy Terraform DSL for the Survey Runner

**How to test**
1. Visit the Preprod jenkins, you can check the jobs exists. I've run them there.

**Who can review**
Anyone apart from @warrenbailey
